src/components/index-page/hero/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { StripesDecoration } from "@/app/conf/_design-system/stripes-decoration"
+import CheckIcon from "@/app/conf/_design-system/pixelarticons/check.svg?svgr"
+import { Button } from "@/app/conf/_design-system/button"
+import { ImageLoaded } from "@/app/conf/2025/components/image-loaded"
+
+import logoBlurred from "./logo-blurred.webp"
+import Head from "next/head"
+
+export function Hero() {
+  return (
+    <div className="relative bg-neu-50 dark:bg-neu-50/10">
+      <div className="gql-container flex flex-col-reverse lg:grid lg:grid-cols-2">
+        <div className="flex max-w-4xl flex-col justify-center gap-4 p-4 lg:min-h-[800px] xl:gap-8 xl:py-24 xl:pl-24 xl:pr-10">
+          <h1 className="typography-h1 max-w-3xl text-neu-900">
+            The query language for modern APIs
+          </h1>
+
+          <ul className="flex flex-col gap-2">
+            {[
+              "Deliver high-performance user experience at any scale",
+              "Secure and stabilize your API with a strongly typed schema and validated requests",
+              "Reduce communication overhead through efficient, distributed development",
```

Comment:
I'm late to the party but this is a lot of words for my limited brain. Proposal for simplification:

```suggestion
              "High-performance at any scale",
              "Typesafe schemas, secure requests",
              "Frictionless distributed development",
```

My [logaf](https://blog.danlew.net/2020/04/15/the-logaf-scale/) is low and this has probably been bikeshedded already, feel free to discard my comment.

Reply:
I love the shortening
the first sentence though was suppose to talk about creating UI at scale, like data co-location stuff
I wonder how we can capture that in a short sentence?

Reply:
New attempt with `Data driven UI at scale`

```suggestion
              "Typesafe schemas, secure requests",
              "Frictionless distributed development",
              "Data driven UI at scale",
```

I am also moving it last because it's a consequence of the first two IMO. If I'm coming to graphql.org and the first thing I see is about UIs, it would have tripped me up.